apph-front/src/index.tsx

```diff
@@ -1,16 +1,38 @@
 import React from 'react';
 import './index.css';
 import reportWebVitals from './reportWebVitals';
-import { SignIn } from './static/components/SignIn';
 import { createRoot } from 'react-dom/client';
+import App from './App';
+import {
+  Route,
+  Routes,
+  Navigate,
+  BrowserRouter as Router
+} from 'react-router-dom';
+import { SignIn } from './static/components/SignIn';
+import { SignUp } from './static/components/SignUp';
 
 const container = document.getElementById('root');
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 const root = createRoot(container!);
+
+function RouterSwitch() {
+  return (
+    <div>
+      <Routes>
+        <Route path="/" element={<App />} />
+        <Route path="/signIn" element={<SignIn />} />
+        <Route path="/signUp" element={<SignUp />} />
+        <Route path="*" element={<Navigate to="/" replace />} />
+      </Routes>
+    </div>
+  );
+}
+
 root.render(
-  <React.StrictMode>
-    <SignIn />
-  </React.StrictMode>
+  <Router>
+    <RouterSwitch />
+  </Router>
```

Comment:
Laisse le `<React.StrictMode>`
```suggestion
  <React.StrictMode>
    <Router>
      <RouterSwitch />
    </Router>
  </React.StrictMode>
```